### ISSUE TITLE
How do I configure and exclude links in MP4 format?

### ISSUE THREAD

**Issue by zooyoyo** [webp-sh/webp_server_go#322]
**Describe the bug**
{        
    "HOST": "0.0.0.0", 
    "PORT": "3333", 
    "QUALITY": "80",  
    "MAX_JOB_COUNT": "10",      
    "IMG_PATH": "http://10.0.2.189:10001",        
    "EXHAUST_PATH": "/home/webp_server_go/cache",        
    "ALLOWED_TYPES": ["jpg","png","jpeg","bmp"]
}
When I click on a video link, I am unable to access it properly

**To Reproduce**
Error :  sendfile: file /home/webp_server_go/cache/local/90036d54bb8e9260 not found

**Expected behavior**

**Screenshots and logs**
If applicable, add screenshots and logs to help explain your problem.

**Environment (please complete the following information):**
 - OS: [e.g. Linux]
 - version or branch[e.g. 0.0.3, master]

**Additional context**

**Comment by n0vad3v** [comment]
Thanks for reporting, as from your config, WebP Server Go is listening on port 3333, you've said "When I click on a video link", are you visiting something like `http://<some_ip_or_domain>:3333/path/to/some.mp4`?

Or is there any reverse proxy in front of WebP Server Go?

**Comment by zooyoyo** [comment]
Yes, I used nginx for proxy. My video link is similar to this: http://10.0.2.189/group1/11.mp4?name=11.mp4&download=0
Nginx configuration:
location /group1 {
proxy_pass http://10.0.2.189:3333;
proxy_set_header Host $host; 
proxy_set_header X-Real-IP $remote_addr; 
proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for; 
}

**Comment by n0vad3v** [comment]
Great, maybe you can try `proxy_pass` only images to WebP Server Go(`http://10.0.2.189:3333/`) while other requests `proxy_pass` directly to your source(`http://10.0.2.189:10001/`), an example Nginx config might like this(I haven't tested this config so I'm not sure whether it works as intended)

```
location ~* ^/group1/.*\.(?:jpg|jpeg|gif|png|svg|heic|bmp|nef|webp)$ {
    proxy_pass http://10.0.2.189:3333;
    proxy_set_header Host $host;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
}

location /group1/ {
    proxy_pass http://10.0.2.189:10001;
    proxy_set_header Host $host;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
}
```

**Comment by zooyoyo** [comment]
I tried it and it is correct. Currently, the link can be accessed normally; I need more tests, thank you!

**Comment by n0vad3v** [comment]
Glad it helps, closing this issue now, if you need more help free feel to reopen this.

**Comment by n0vad3v** [comment]
Hi! @zooyoyo 
we've released 0.13.0 that should make WebP Server to handle non-image files by it self(without the need on Nginx).
https://github.com/webp-sh/webp_server_go/releases/tag/0.13.0
https://blog.webp.se/webp-server-0-13-en/

Feel free to give it a try and feedbacks are welcomed.❤️

**Comment by zooyoyo** [comment]
Yes, I tried it, because it's no problem for me
